src/renderer/entities/network/model/network-model.ts

```diff
@@ -69,6 +64,11 @@ const getDefaultStatusesFx = createEffect((chains: Record<ChainId, Chain>): Reco
   return dictionary(Object.values(chains), 'chainId', () => ConnectionStatus.DISCONNECTED);
 });
 
+persist({
+  key: 'chains_map',
+  store: $chains,
```

Comment:
why did you removed sync flag? it indicates that lib should load local storage value on startup, i didn't found any related logic in new code